std/encoding.d

```diff
@@ -2462,7 +2462,7 @@ abstract class EncodingScheme
      * Params:
      *    s = the array to be tested
      */
-    bool isValid(const(ubyte)[] s)
+    bool isValid()(const(ubyte)[] s)
```

Comment:
Especially where this is an abstract class and is *meant* to be overridden.

@JackStouffer inference is not really needed for simple signatures like this, especially when no templates are involved. Just mark them explicitly.